### DISCUSSION TITLE
The system could not find the environment option that was entered

### DISCUSSION THREAD

**Discussion by mrdo23** [hardkoded/puppeteer-sharp#1942] [Q&A]
Hi,

Attempting to get puppeteersharp running on a windows-hosted azure function. 

Have pre-downloaded the exe and have copied the 'chrome-win' directory to the wwwroot directory for use by the function.

Am getting 'The system could not find the environment option that was entered.System.ComponentModel.Win32Exception' when calling Puppeteer.LaunchAsync with the following params: Headless = true, ExecutablePath = path where the path the verified location of the exe: 'D:\home\site\wwwroot\TEST\chrome-win\chrome.exe'.

Does Puppeteer use an environment setting in windows that isn't being set correctly because I'm not downloading in the usual way?

**Comment by naulacambra** [comment]
I'm getting the same error, but I've let Puppeteer download the chrome. Did you fixed it?

**Comment by mrdo23** [comment]
I gave up completely on Windows - not sure if it was ever going to work. Unfortunately, when moving to Linux I had a whole new set of issues with Puppeteer not displaying Asian fonts due to an underlying Chrome issue. Have now moved to a Linux Docker image and STILL having problems! Good luck.
